### ISSUE TITLE
dashboards: Split example dashboard into tiers (GatewayClass > Gateway > *Route) with filters

### ISSUE THREAD

**Issue by david-martin** [Kuadrant/gateway-api-state-metrics#7]
The current dashboard shows GatewayClasses, Gateways and HttpRoutes all in the 1 view.

To facilitate navigation and filtering:
- split this into 3 separate dashboards
- Add filter templates/dropdowns to each dashboard to allow filtering by appropriate resources e.g.
  - filter HTTPRoutes by targetted Gateway when viewing the HTTPRoute dashboard
  - filter Gateways by gateway name
- Add drill down links to aid navigation between the tiered dashboards
- If done after #3, allow drilling down and filtering for those Route types as well
- Name the dashboards consistently and place in a folder. Suggestion is a folder called 'Gateway API'
